terraform/workspace_variables/preprod.tfvars.json

```diff
@@ -4,5 +4,13 @@
   "resource_group_name": "s165t01-preprod-rg",
   "flt_app_name": "find-a-lost-trn-preprod",
   "paas_space": "tra-test",
-  "postgres_database_name": "find-a-lost-trn-preprod-pg-svc"
+  "postgres_database_name": "find-a-lost-trn-preprod-pg-svc",
+  "statuscake_alerts": {
+    "tra-flt-preprod": {
+      "website_name": "find-a-lost-trn-preprod",
+      "website_url": "https://find-a-lost-trn-preprod.london.cloudapps.digital/health",
+      "test_type": "HTTP",
+      "contact_group": ["twd_tra_dqt"]
```

Comment:
We tried using the contact group name a few months ago with register.  The same terraform resource version used here but it didn't work.  It seems to only work with the contact group id as shown in the [example](https://github.com/DFE-Digital/find-a-lost-trn/pull/81) though the argument reference is a bit ambiguous.

Reply:
Thats strange. I have used the contact group name for [DQT API](https://github.com/DFE-Digital/qualified-teachers-api/blob/main/terraform/preprod.tfvars.json) before. Unless something changed with the provider recently. Let me try deploying and see.